etc/kayobe/inventory/group_vars/controllers/network

```diff
@@ -0,0 +1,28 @@
+# List of IP routing tables. Each item should be a dict containing 'id' and
+# 'name' items. These tables will be added to /etc/iproute2/rt_tables.
+network_route_tables:
+  # IP routing table for traffic orginating from the public network. Because of
+  # the asymmetric routing in the system, we use policy-based routing to route
+  # based on the source address and the destination address for the public
+  # network.
+  - name: public
+    id: 1
+
+# Public network IP routing rules.
+public_rules:
+  # Use the public routing table for all traffic originating from the public
+  # network.
+  - "from {{ public_cidr }} table public"
+
+# Public network static IP routes.
+public_routes:
+  # Default route via the firewall's public network interface, added to the
+  # public routing table.
+  - cidr: "0.0.0.0/0"
+    gateway: "10.12.9.1"
```

Comment:
Is this gateway IP parameterised anywhere?

Reply:
Not that I am aware of.